tests/integration/test_federation.py

```diff
@@ -7,6 +7,7 @@
 from contextlib import contextmanager
 from bigchaindb.common.crypto import generate_key_pair
 from tests.pipelines.stepping import create_stepper
+from tests.conftest import skip_travis_rdb
```

Comment:
Is this import really needed?

Reply:
Is there another way I can access `skip_travis_rdb` in this module?

Reply:
@sbellem I think I can directly do this as `@pytest.mark.usefixtures("skip_travis_rdb")`